ui/src/main.rs

```diff
@@ -85,8 +89,10 @@ pub static STATIC_ARGS: Lazy<StaticArgs> = Lazy::new(|| {
     };
     StaticArgs {
         uplink_path: uplink_path.clone(),
+        // TODO: This isn't quite accurate since uplink path is technically what we consider the cache dir.
```

Comment:
sounds like `.uplink` should have a sub-directory for the cache? that would definitely help when overwriting an account - it would let the `themes` directory not get deleted.